pygmt/tests/test_config.py

```diff
@@ -36,10 +37,7 @@ def test_config():
     return fig
 
 
-@pytest.mark.xfail(
-    reason="Baseline image not updated to use earth relief grid in GMT 6.1.0",
-)
-@pytest.mark.mpl_image_compare
+@check_figures_equal()
 def test_config_font_one():
```

Comment:
This test doesn't make sense to me.

We should check if `with config(FONT="8p,red"):` has the same effects as
```
with config(FONT_ANNOT_PRIMARY="8p,red", FONT_LABEL="8p,red", FONT_TITLE="8p,red", ...):
```

Reply:
Great suggestion, I've made the change in eeccc77220bfba2b354eb00f2f073c5b6952f9c1.